saagieapi/env_vars/gql_queries.py

```diff
@@ -64,6 +57,66 @@
         value
         description
         isPassword
+        isValid
```

Comment:
The doc of the section `Listing project environment variables` should be updated